test/lint/checkStyle.js

```diff
@@ -0,0 +1,19 @@
+var fs = require('fs')
+var jsonDiff = require('./jsonDiff')
+
+function checkStyle (filename) {
+  var noErrors = true
+  var actual = fs.readFileSync(filename, 'utf-8').trim()
+  var expected = JSON.stringify(JSON.parse(actual), null, 2)
+
+  if (actual === expected) {
+    console.log('  Style – OK')
+  } else {
+    noErrors = false
+    console.log('  Style – Error on line ' + jsonDiff(actual, expected))
```

Comment:
It would be nice maybe if this displayed all the diffs rather than just one line at the time.

Reply:
It's ok, `jsonDiff` returns all the diffs.